src/main/java/ure/actors/UActorCzar.java

```diff
@@ -36,7 +36,7 @@ public void loadActors(String resourceName) {
     }
 
     public UActor getActorByName(String name) {
-        return (UActor)actorsByName.get(name).getClone();
+        return (UActor)actorsByName.get(name).makeClone();

```

Comment:
Using the `get` prefixes for things that don't actually have matching fields confuses Jackson 😞